modules/mobile/config/initializers/statsd.rb

```diff
@@ -24,15 +24,6 @@
 
   # Appointments #---------------------------------------------------------------
 
-  # meta binding
```

Comment:
Should we change this to the V2 proxy or is this no longer necessary?

Reply:
Ideally this should be tracked in datadog but right now appointments are lumped all into one metric. I know I'm the one that initially suggested to delete it but now I'm thinking we should change to v2 until it shows in datadog properly. if we end up deleting though we should delete the next 4 lines too as they initialize the metrics that are getting deleted.

Reply:
@kpethtel Do you have any thoughts on this?

Reply:
Yeah, i think andrew is probably right. It seems like the v2 metrics were overlooked somehow. We should probably do essentially redo these metrics for v2. Deleting these in this ticket seems fine, but I'd suggest adding the new ones in a separate ticket.

Reply:
Ah it should be a separate ticket? I just pushed an update that adds it in this ticket because I figured it was "transitioning" from v0 to v2. We should be completely done with v0 to v2 after this

Reply:
Just a suggestion. NBD.

Reply:
Ok if you're cool I'd just like to change it here since it's the same lines of code changed and we can put a wrap on the v0 to v2 transition